tests/cephadm/test_remove_cluster.py

```diff
@@ -0,0 +1,111 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CephadmOpsFailure(Exception):
+    pass
+
+
+def _disable_mgr_module(node):
+    """Disable cephadm module"""
+    cmd = "cephadm shell -- ceph mgr module disable cephadm"
```

Comment:
If `mgr` object or method is unavailable can we implement the module and leverage the same.

The purpose of `cephadm` object is to act as an interface to perform CLI operations. The current implementation does not support re-usability

Reply:
Currently `ceph/ceph_admin/manager.py` module has some operations related to `mgr` object. Will try to convert this module into `ManagerMixin` and use it.